### ISSUE TITLE
Remote installation of the soma-workflow server is incompletely configured

### ISSUE THREAD

**Issue by ylep** [populse/soma-workflow#56]
In the GUI, under *Server management* / *Add Server*, the “Install on Server and Client” action fails for me: while the soma-workflow code is copied successfully on the server, the `PYTHONPATH` fails to be correctly configured:

1. The `export PYTHONPATH=...` line is never added to my `~/.bashrc`
2. Even if I add it manually, it still does not work because `~/.bashrc` is only parsed when a full-featured remote shell is launched, **not** when a remote command is executed through `paramiko` (similarly with `ssh user@host printenv PYTHONPATH`).

Moreover, wouldn't it make more sense to store the server’s PYTHONPATH in the client configuration, and state it explicitly with each remote command, so that it is only used by this specific client, rather than imposing it globally in `~/.bashrc`? I suspect that the `~/.bashrc` approach can very easily lead to version mismatches.

@populse/soma we need to reach an agreement on how to proceed with this issue, what are your thoughts?

**Comment by denisri** [comment]
Yes I agree and this is one of the reasons I was not completely fan of this auto-install feature, but on the other side it is useful for people who need to quickly setup soma-workflow.
Another way is an install via pip on server side.
